examples/digitalidentity/app/Http/Controllers/IdentityController.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Routing\Controller as BaseController;
+use Illuminate\Support\Facades\Log;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+use Yoti\DigitalIdentityClient;
+use Yoti\Identity\Policy\PolicyBuilder;
+use Yoti\Identity\ShareSessionRequestBuilder;
+use Yoti\YotiClient;
+
+class IdentityController extends BaseController
+{
+    public function show(DigitalIdentityClient $client)
+    {
+        try {
+
+            $policy = (new PolicyBuilder())
+                ->withFamilyName()
+                ->withGivenNames()
+                ->withFullName()
+                ->withDateOfBirth()
+                ->withGender()
+                ->withNationality()
+                ->withPhoneNumber()
+                ->withSelfie()
+                ->withEmail()
+                ->withDocumentDetails()
+                ->withDocumentImages()
+                ->build();
+
+            $redirectUri = 'https://host/redirect/';
```

Comment:
Please use a real path in the example for redirection.

Reply:
Changed to real url

Reply:
changed to real url